graql/reasoner/type-generation.feature

```diff
@@ -29,6 +29,8 @@ Feature: Type Generation Resolution
 
 
   # TODO: re-enable all steps once attribute re-attachment is resolvable
+  #  TODO: disabled because down and side casting no longer allowed
```

Comment:
Same comment as above - if we're settled on banning type generation, we should delete these scenarios entirely

Reply:
deleted